modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -173,20 +169,30 @@ async function invokeServer<T>(
 					}),
 					method: 'POST',
 				})
-					.then((response) => response.text())
-					.then((text) => {
-						const json = JSON.parse(text);
-
-						if (json.error) {
-							resolve({
-								error: json.error,
-							});
-						}
-						else {
-							resolve({
-								result: json,
-							});
+					.then(async (response) => {
+						if (!response?.ok) {
+							if (response?.status === 404) {
+								reject(
+									`Feature flag "${partialBody['key']}" is not available`
```

Comment:
`partialBody['key']` usually we would prefer `partialBody.key`

Reply:
done

Reply:
oh, actually it broke the sf. It seems it needs to be this way

Reply:
done